mix/qml/DebugBasicInfo.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.2
+import QtQuick.Controls 1.1
+import QtQuick.Layouts 1.0
+import QtQuick.Controls.Styles 1.1
+
+ColumnLayout {
+	property string currentStep
+	property string mem
+	property string stepCost
+	property string gasSpent
+	spacing: 0
+	RowLayout {
```

Comment:
​I have still some changes to make​, This pull request is still in "In
Progress". I think I will change to "Please Review" as soon as Alex will
approve the design implementation (And also as soon I have cleaned the
code).

2015-01-08 11:53 GMT-08:00 Arkadiy Paronyan <notifications@github.com>:

> In mix/qml/DebugBasicInfo.qml
> <https://github.com/ethereum/cpp-ethereum/pull/757#discussion-diff-22676753>
> :
>
> > @@ -0,0 +1,109 @@
> > +import QtQuick 2.2
> > +import QtQuick.Controls 1.1
> > +import QtQuick.Layouts 1.0
> > +import QtQuick.Controls.Styles 1.1
> > +
> > +ColumnLayout {
> > +	property string currentStep
> > +	property string mem
> > +	property string stepCost
> > +	property string gasSpent
> > +	spacing: 0
> > +	RowLayout {
>
> Repeating elemets should be defined as components to avoid copy-pasting
> blocks of similair code
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/cpp-ethereum/pull/757/files#r22676753>.
>